infrastructure/validators/offboarding/stop-validating.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Stop Validating
+description: Instructions on how to stop validating.
+---
+
+If you wish to remain a validator or nominator (e.g. you're only stopping for planned downtime or
+server maintenance), submitting the `chill` extrinsic in the `staking` pallet should suffice. It is
+only if you wish to unbond funds or reap an account that you should continue with the following.
+
+To ensure a smooth stop to validation, make sure you should do the following actions:
+
+- Chill your validator
+- Purge validator session keys
+- Unbond your tokens
+
+These can all be done with [PolkadotJS Apps](https://polkadot.js.org/apps){target=_blank} interface or with
+extrinsics.
+
+
+<!-- TODO: add links later -->
+
+## Chill Validator
+
+To chill your validator or nominator, call the `staking.chill()` extrinsic. See the How to Chill page for more information. You can also claim your rewards at this time.
+
+## Purge validator session keys
+
+Purging the validator's session keys removes the key reference. This can be done through the
+`session.purgeKeys()` extrinsic. The key reference exists on the account that originally called the
+`session.set_keys()` extrinsic, which could be the stash or the staking proxy (at the time the keys
+were set).
+
+!!!warning "Purge keys using the same account that set the keys"
+    Make sure to call the session.purge_keys() extrinsic from the same account that set the keys in the
+    first place in order for the correct reference to be removed. Calling the `session.purge_keys()`
+    from the wrong account, although it may succeed, will result in a reference on the other account
+    that **cannot** be removed, and as a result that account cannot be reaped.
+
+!!!warning
+    **If you skip this step, you will not be able to reap your stash account**, and you will also need
```

Comment:
```suggestion
    **If you skip this step, you won't be able to reap your stash account**, and you will also need
```
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.